controllers/usersController.js

```
@@ -1,21 +1,38 @@
-const express = require('express');
-const { getAllUsers, getUserById } = require('../queries/usersQueries');
+const express = require("express");
+const { getAllUsers, getUserById } = require("../queries/usersQueries");
 
 const usersController = express.Router();
 
-usersController.get('/', (req, res) => {
-  users = getAllUsers() || [];
-  res.status(200).json({ data: users });
+// /users
+usersController.get("/", (req, res) => {
+  getAllUsers()
+    .then((users) => {
+      res.status(200).json({ data: users });
+    })
+    .catch((err) => {
+      console.log(err)
+      res.status(500).json({ error: "An error has ocurred when tries to get all users" })
```

This result should include `error.message` instead of a default error message. We always want to return the actual error to the client so that they know what exactly went wrong. This makes it easier for clients to fix their mistakes and also makes it easier for developers to debug the service.